video/profiles.go

```diff
@@ -104,10 +109,10 @@ func GetPlaybackProfiles(iv InputVideo) ([]EncodedProfile, error) {
 
 func lowBitrateProfile(video InputTrack) EncodedProfile {
```

Comment:
This method's getting (slash already way) pretty complicated to follow - could you add a bit of a descriptive comment please? i.e why do we halve the bitrate to begin with